cpp/plugins/cucim.kit.cuslide/src/cuslide/jpeg/libjpeg_turbo.cpp

```diff
@@ -196,6 +231,119 @@ bool read_jpeg_header_tables(const void* handle, const void* jpeg_buf, unsigned
     return true;
 }
 
+// tjDecompress2
+int jpeg_decode_buffer(const void* handle,
+                       const unsigned char* jpegBuf,
+                       unsigned long jpegSize,
+                       unsigned char* dstBuf,
+                       int width,
+                       int pitch,
+                       int height,
+                       int pixelFormat,
+                       int flags,
+                       int jpegColorSpace)
+{
+    JSAMPROW* row_pointer = NULL;
+    int i, retval = 0, jpegwidth, jpegheight, scaledw, scaledh;
+
+    tjinstance* instance = (tjinstance*)handle;
+    j_decompress_ptr dinfo = NULL;
+    dinfo = &instance->dinfo;
+    instance->jerr.warning = FALSE;
+    instance->isInstanceError = FALSE;
+
+    instance->jerr.stopOnWarning = (flags & TJFLAG_STOPONWARNING) ? TRUE : FALSE;
```

Comment:
Should we use `true` & `false` for these?

```suggestion
    instance->jerr.stopOnWarning = (flags & TJFLAG_STOPONWARNING) ? true : false;
```

Reply:
Same reason for above.

TRUE/FALSE is defined by `build-debug/_deps/deps-libjpeg-turbo-src/jmorecfg.h` which is included by <jpeglib.h> in this file.